### DISCUSSION TITLE
Whether the go-mod-core-contracts and go-mod-messaging in edgex is stable?

### DISCUSSION THREAD

**Discussion by LavenderQAQ** [edgexfoundry/community#155] [General]
Hi everyone, I am a maintainer of the OpenYurt community and we have plans to integrate some of our components with edgex using messagebus.

We found that `go-mod-messaging` packages the usage of messagebus, and `go-mod-core-contracts` is used for topic construction. At present, we will also call these two libraries for development, to get some information in messagebus.

I would like to ask if the edgex community will maintain the long-term stability of these two libraries, and if there are some disruptive changes to these two libraries in the future, it will have a significant impact on the features we want to develop.

**Comment by LavenderQAQ** [comment]
refer: https://github.com/openyurtio/yurt-device-controller/issues/60

**Comment by bnevis-i** [comment]
See the EdgeX Wiki for the [long term support policy](https://wiki.edgexfoundry.org/x/VIAfB).

In general, you can expect go-mod-core-contracts/v2, etc to be backwards-compatible with itself and go-mod-core-contracts/v3, etc to contain breaking changes relative to the previous major version.  Plain go-mod-core-contracts is interpreted to go as go-mod-core-contracts/v1.
